deployment/scripts/gen-deploy.sh

```diff
@@ -16,7 +16,7 @@ gen() {
 
 	mkdir -p "$(dirname ${OUTPUT})"
   cp chart/keel/values.yaml "${TMP_VALUES}"/
-  sed -i "" 's/false/true/g' "${TMP_VALUES}/values.yaml"
+  sed -i 's/false/true/g' "${TMP_VALUES}/values.yaml"
```

Comment:
not if you do this to the values.yaml ;)

Reply:
perhaps it's better to omit the sed altogether and just enable the values with `--set` in the helm cli directly.

But as I said, the only ones that might work out-of-the-box seems to be `debug`, `podDisruptionBudget` and `service`.

Shall I put those in & see what happens?

Reply:
I haven't been using helm for quite some time so not sure how it should work. What do you think @rimusz ? Maybe try generating it and see whether sidecar container still remains there?